docs/index.md

```diff
@@ -9,14 +9,80 @@ The [CLI](cli-reference/dub.md) can be used to
 - compile projects and external programs ([dub build](cli-reference/dub-build.md), [dub run](cli-reference/dub-run.md))
 - test projects ([dub test](cli-reference/dub-test.md))
 
-To see how to obtain dub, **go to the next page by pressing the button below**.
+DUB is bundled with most D compilers' distributions. However, it's also possible to install it separately.
+See [Installing DUB](getting-started/install.md) for details.
 
 <!-- old docs anchors for index page, all link to first steps -->
 <a id="own-projects"></a>
 <a id="adding-deps"></a>
 <a id="foreign-projects"></a>
 <a id="advanced-usage"></a>
 
-## Starting a new project
+## DUB Basics
 
-You can also skip ahead to [First steps](getting-started/first-steps.md) if you already have dub installed.
+DUB is a build tool similar to other modern languages build tools like Javascript's [npm](https://www.npmjs.com/)
+and Rust's [cargo](https://crates.io/).
+
+A file called `dub.sdl` (or `dub.json`) is used to configure a DUB project.
```

Comment:
Technically, that is called the "recipe file"

Reply:
I would mention the 3 files that drive dub:
- Recipe file, per project;
- Selections file, or lock file, tied to a recipe file;
- Configuration file, which can be at project, user, or system scope;

Reply:
Configuration file refers to this one? https://dub.pm/dub-reference/settings/

Reply:
I've edited the text to mention all three files.